exercises/acronym/.meta/tests.toml

```diff
@@ -0,0 +1,28 @@
+[canonical-tests]
```

Comment:
This and the tests.toml file below seem unrelated to the PR. Merging probably won't cause any issues, just pointing it out in case they were committed by accident.